lib/map.rs

```diff
@@ -0,0 +1,11 @@
+use crate::closure::Closure;
+use crate::ddlog_std;
+
+pub fn map_map_in_place<K: Ord, V>(
+    m: &mut ddlog_std::Map<K, V>,
+    f: &Box<dyn Closure<(*const K, *mut V), ()>>,
```

Comment:
Do we document somewhere the relationship between ddlog types and rust types for extern function writers?

Reply:
Not at the moment, but we should.